src/lidar/bg_flood_model.py

```diff
@@ -109,40 +112,44 @@ class BGDEM(Base):
 
 
 def run_model(
-    bg_path,
-    instructions,
-    catchment_boundary,
-    resolution,
-    endtime,
-    outputtimestep,
-    engine,
+        bg_path,
+        instructions,
+        catchment_boundary,
+        resolution,
+        endtime,
+        outputtimestep,
+        engine,
 ):
     """Call the functions."""
     dem_path = dem_metadata_in_db.get_dem_path(instructions, engine)
     bg_model_inputs(
         bg_path, dem_path, catchment_boundary, resolution, endtime, outputtimestep
     )
     os.chdir(rf"{bg_path}")
-    subprocess.call([rf"{bg_path}\BG_Flood_Cleanup.exe"])
+    subprocess.call([rf"{bg_path}/BG_Flood_Cleanup.exe"])
```

Comment:
Same as above RE moving to using pathlib for future PRs

Reply:
done